docs/tutorial/sql.md

```diff
@@ -78,6 +78,23 @@ The output will be like this:
 |POLYGON ((-104.56...| 35|011|00933054|35011|    De Baca|      De Baca County| 06| H1|G4020|null| null|null|   A|6015539696|29159492|+34.3592729|-104.3686961|
 |POLYGON ((-96.910...| 31|109|00835876|31109|  Lancaster|    Lancaster County| 06| H1|G4020| 339|30700|null|   A|2169240202|22877180|+40.7835474|-096.6886584|
 ```
+## Load data from shape file directly into DataFrame
```

Comment:
Can you please remove the change on docs for now? I am thinking to have a more general API to load DataFrame fields names for different inputs  such as shapefile and GeoJSON.